File: legacy/legacy-apis/Psi.Models.ClientConfigurationModels/Themes/Themes.cs
namespace Psi.Data.Models.ClientConfigurationModels.Themes
{
    public class Themes : SettingsBaseHelper
    {
	    private ConnectNative _connectNative;
	    private ISettingsBase _settingsBase;

		public Themes(ISettingsBase settingsBase) : base(settingsBase)
		{
			_settingsBase = settingsBase;
		}

	    public ConnectNative ConnectNative
		{
		    get { return _connectNative ?? (_connectNative = new ConnectNative(_settingsBase)); }
		    set { _connectNative = value; }
	    }
    }
}

File: infrastructure/src/models/Psi/Data/Models/ClientConfigurationModels/Themes/Themes.ts
import { Setting, ISettingsGroup, ISettingsMetadata } from '@models/base/types';
import { ConnectNative } from './ConnectNative';
export interface ThemesConfig {
    ConnectNative: ConnectNative;
}

export class Themes implements ISettingsGroup {
    static readonly metadata: ISettingsMetadata = {
        groupName: 'Themes'
    };


            private _connectNative: ConnectNative;
            get connectNative(): ConnectNative {
                return this._connectNative;
            }
            set connectNative(value: ConnectNative) {
                this._connectNative = value;
            }


    constructor() {}


        toSettings(): Setting[] {
            return [
                { key: "Themes.ConnectNative", value: this._connectNative, dataType: 'connectnative', label: "Connect Native" },
            ];
        }

}